src/modules/graphics/Graphics.cpp

```diff
@@ -359,6 +368,9 @@ Shader *Graphics::newShader(const std::vector<std::string> &stagessource, const
 	for (const std::string &source : stagessource)
 	{
 		Shader::SourceInfo info = Shader::getSourceInfo(source);
+		if (capabilities.features[FEATURE_GLSL4]) {
```

Comment:
Just making a note here: I think we should move this to either the same place as `LOVE_GLSL1_ON_GLSL3`, or inside the vulkan shader compilation code.

Reply:
Turns out this code is uneeded anyway. I think when I was trying to get shaders working the first time it was a necessary hack. I'll remove it.